requirements.txt

```diff
@@ -1,2 +1,3 @@
-projectq
+git+https://github.com/ProjectQ-Framework/ProjectQ.git#egg=projectq
```

Comment:
Do we want to change this back to stable?

Reply:
Yes I think so!